app/uk/gov/hmrc/helptosavestub/controllers/EligibilityCheckController.scala

```diff
@@ -16,19 +16,20 @@
 
 package uk.gov.hmrc.helptosavestub.controllers
 
-import org.scalacheck.Gen
 import play.api.libs.json.{Format, Json}
 import play.api.mvc._
-import uk.gov.hmrc.play.microservice.controller.BaseController
-import hmrc.smartstub._
 import uk.gov.hmrc.helptosavestub.controllers.EligibilityCheckController.EligibilityCheckResult
+import uk.gov.hmrc.play.microservice.controller.BaseController
+
+import scala.util.Try
 
 class EligibilityCheckController extends BaseController {
 
   val resultMappings: Map[Int, String] = Map(
     1 → "Eligible to HtS Account",
     2 → "Ineligible to HtS Account",
-    3 → "HtS account already exists"
+    3 → "HtS account already exists",
+    4 → "INVALID RESULT WHICH DES SHOULD NEVER SEND"
```

Comment:
i am not comfortable adding an invalid code here

Reply:
I'd like the ability to check that we handle incorrect values from DES gracefully.

Reply:
the incorrect values are handled in the frontend accordingly